packages/data/lib/mapper/mapper.dart

```diff
@@ -0,0 +1,5 @@
+abstract class Mapper {
```

Comment:
good idea, but not the best implementation - u should create `abstract class Mapper<I, O>`. I - input type, O - output type. this class should have method call - `O call(I params)`. then create subclass, `IsPalindromeMapper implements Mapper<Map<String, dynamic>, bool>`, override method call. and then create the mapper object and use it anywhere u need

Reply:
delete class